core/templates/rid_owner.h

```diff
@@ -419,8 +416,8 @@ class RID_PtrOwner {
 		alloc.set_description(p_descrption);
 	}
 
-	RID_PtrOwner(uint32_t p_target_chunk_byte_size = 65536) :
-			alloc(p_target_chunk_byte_size) {}
+	RID_PtrOwner(uint32_t p_target_chunk_byte_size = 65536, uint32_t p_maximum_amount_of_elements = 262144) :
```

Comment:
```suggestion
	RID_PtrOwner(uint32_t p_target_chunk_byte_size = 65536, uint32_t p_maximum_number_of_elements = 262144) :
```
"Number" for countable things

Reply:
Done.